usagov/usagov-2021
# USAGOV-1624 Set default value in bootstrap.sh

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-

## Description
Sets CF_SYSTEM_CERT_PATH variable to an empty string if it's unset.

## Type of Changes

- [ X ] Bugfix
  - Add screenshot showing what it should look like
- [ X ] Infrastructure
  - [ X ] Tools

## Testing Instructions


Unset CF_SYSTEM_CERT_PATH in a local environment and run bootstrap.sh and see if it still executes.

### Requires New Config
- [ X ] No

### Requires New Content
- [ X ] No

### Validation Steps
1. In a local environment, bin/ssh.
2. Run bootstrap.sh
3. The script runs as expected.
4. Unset CF_SYSTEM_CERT_PATH in a local environment by reverting the changed lines to:
"if [ -d "$CF_SYSTEM_CERT_PATH" ]; then
   cp $CF_SYSTEM_CERT_PATH/*  /usr/local/share/ca-certificates/
fi"
5. Run bootstrap.sh to see if it still executes.
## Security Review


- [ X ] Adds/updates software (including a library or Drupal module)


## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
5. Find the commit of this pull request.
6. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
